### PR TITLE
Web3.js Ext AccountKey

### PR DESCRIPTION
**Pull request by nohkwak**
Add examples and Test each cases. 

- AccountKeyLegacy : ValueTransfer 
- AccountKeyPublic : AcountUpdate, ValueTransfer 
- AccountKeyWeightedMultiSig : AcountUpdate, ValueTransfer 
- AccountKeyRoleBased : AcountUpdate, ValueTransfer

### CODE REVIEW

File: web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_02_valueTransfer.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    type: TxType.ValueTransfer,
+    gasLimit: 100000,
+    to: receiverAddr,
+    value: 1e9,
+    // value: convertToPeb('1', 'KLAY'),
+    from: senderAddr,
+  };
+
+  // sign 1
+  const wallet = web3.eth.accounts.privateKeyToAccount(senderNewPriv1);
```
Comment (kjeom) [inline web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_02_valueTransfer.js:34]: It's minor review.
wallet variable name isn't fit for web3js.
it's better to make it just as 'account' or 'acc'.

File: web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_01_accountUpdate.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const senderPriv = "0x7394141c6fa82980656212a8d15f120e8f1ac81e6705a49864e86f6c507f239e";
+const senderNewPriv1 = "0xa32c30608667d43be2d652bede413f12a649dd1be93440878e7f712d51a6768a";
+const senderNewPriv2 = "0x0e4ca6d38096ad99324de0dde108587e5d7c600165ae4cd6c2462c597458c2b8";
+const senderNewPriv3 = "0xc9668ccd35fc20587aa37a48838b48ccc13cf14dd74c8999dd6a480212d5f7ac";
+
+async function main() {
+  const provider = new Web3.providers.HttpProvider("https://public-en-baobab.klaytn.net");
+  const web3 = new KlaytnWeb3(provider);
+
+  const senderNewPub1 = "0x" + Buffer.from(secp256k1.getPublicKey( BigInt(senderNewPriv1), true)).toString('hex');
```
Comment (kjeom) [inline web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_01_accountUpdate.js:27]: Could you make a util function for it?

Comment (blukat29) [reply web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_01_accountUpdate.js:27]: Let's fix that with js-ext-core.

**Review by blukat29 — APPROVED** [review]

File: web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_02_valueTransfer.js (hunk 1 above)
Comment (nohkwak) [reply web3js-ext/example/accountKey/AccountKeyWeightedMultiSig_02_valueTransfer.js]: Thanks. 
Updated in https://github.com/klaytn/web3klaytn/pull/520/commits/1cc4c6a8e571a07e104997294114ada72481de3e.

**Review by blukat29 — APPROVED** [review]

**Review by blukat29 — APPROVED** [review]

**Pull request merged**